vimrc

```diff
@@ -0,0 +1,54 @@
+"Load the plugins before trying to configure any settings
+source ~/.vim/install-plugins.vim
+source ~/.vim/airline-config.vim
+source ~/.vim/syntastic-config.vim
+source ~/.vim/plugin-config.vim
+source ~/.vim/nerdtree-config.vim
```

Comment:
A few of these sources should really be part of the `plugin-config.vim` file instead.